roles/sft-server/templates/sftd.unit.j2

```diff
@@ -0,0 +1,27 @@
+[Unit]
+Description=Running service: SFT server
+
+After=nginx.service
+Wants=nginx.service
+
+
+[Service]
+TimeoutStartSec=0
+Restart=always
+
+# maximum size of core files (bytes)
+LimitCORE=infinity
+LimitCORESoft=0
```

Comment:
Don't you also need LimitNOFILE ? Otherwise you're restricted to 1024 connections be default, no?

Reply:
Absolutely right! Added.